v2.6/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -3,141 +3,101 @@ title: Going Beyond NetworkPolicy with Calico
 redirect_from: latest/getting-started/kubernetes/tutorials/advanced-policy
 ---
 
-The Kubernetes NetworkPolicy API allows users to express ingress policy to Kubernetes pods
-based on labels and ports.  Calico implements this API, but also supports a number of
-policy features which are not currently expressble through the NetworkPolicy API such as CIDR
-and egress policy.
+The Kubernetes NetworkPolicy API allows users to express ingress and egress policies (starting with K8s 1.8.0) to Kubernetes pods
```

Comment:
These are minor nits but it would really help us out if you could make a few other changes:

- Here in the body is actually the reverse, where backticks work, and we are looking to standardize on using the backticks to enclose "NetworkPolicy" when in camel case like this. So, if you could add backticks around "NetworkPolicy" that would be amazing, e.g., `NetworkPolicy`. (Throughout)

- Also, this is a minor nit, but if you could spell out Kubernetes instead of using "K8s" that would be much, much better. Also, as a proper noun, kubernetes should always be capitalized. (Throughout)